app/services/admin_notices/dismiss.rb

```diff
@@ -3,25 +3,23 @@
 class AdminNotices::Dismiss
   include Service::Base
 
-  model :admin_notice, optional: true
-
   policy :invalid_access
```

Comment:
Moving this policy to the top as it only relies on `guardian` and will avoid having to fetch a model for nothing.